src/components/AnonymousReportFooter.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import PropTypes from 'prop-types';
+import Button from './Button';
+import AvatarWithDisplayName from './AvatarWithDisplayName';
+import ExpensifyWordmark from './ExpensifyWordmark';
+import compose from '../libs/compose';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import reportPropTypes from '../pages/reportPropTypes';
+import CONST from '../CONST';
+import styles from '../styles/styles';
+import * as Session from '../libs/actions/Session';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: reportPropTypes,
+
+    /** Type of authToken for the logged in user */
+    authTokenType: PropTypes.string,
+
+    ...windowDimensionsPropTypes,
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    report: null,
```

Comment:
I think this might be an issue in [AvatarWithDisplayName](https://github.com/Expensify/App/blob/109421eb954a76c78db04abe1e4ac216a433d94c/src/components/AvatarWithDisplayName.js#L48) too, because we'll try to access `props.report.ownerEmail` on a null type, which will throw.

```suggestion
    report: {},
```

Reply:
Oh good catch, updating it